packages/valory/skills/abstract_round_abci/models.py

```diff
@@ -277,8 +283,20 @@ def _get_response_data(
 
             return self._get_response_from_index(value)
 
-        except (json.JSONDecodeError, KeyError, IndexError):
-            return None
+        except json.JSONDecodeError:
+            self.context.logger.error("Could not parse the response body!")
+        except KeyError:
+            self.context.logger.error(
+                f"Could not access response using the given key(s) ({self.response_key})!"
+            )
+        except IndexError:
+            self.context.logger.error(
+                f"Could not access response using the given key(s) ({self.response_key}) "
+                f"and index ({self.response_index})!"
+            )
+
+        self._log_response(decoded_response)
```

Comment:
This will log even when the response is not a valid JSON, was that intended?

Reply:
Yes, it was, so that the users have all the information they need to fix their configuration or understand why the response could not be accessed.